README.md

```diff
@@ -178,6 +178,46 @@ helm install <release-name> -f <values.yaml> anchore/k8s-inventory
 
 A basic values file can always be found [here](https://github.com/anchore/anchore-charts/tree/main/stable/k8s-inventory/values.yaml)
 
+## The anchore-k8s-inventory agent as an Enterprise Integration
+Starting from release `v5.11.0`, Enterprise provides a new registration and health reporting feature whereby agents
```

Comment:
In the Enterprise v5.11.0 release, Enterprise provides the ability for k8s-inventory to register and provide health reporting.